packages/xo-server/src/xapi/mixins/patching.mjs

```diff
@@ -548,7 +553,15 @@ export default {
       await this._waitObjectState(metricsRef, metrics => metrics.live)
       const rebootTime = parseDateTime(await this.call('host.get_servertime', host.$ref))
 
-      log.debug(`Evacuate and restart host ${hostId}`)
+      if (isXcp) {
+        // On XCP-ng, install patches on each host one by one instead of all at once
+        log.debug(`Evacuate host ${hostId}`)
+        await this.clearHost(host)
+        log.debug(`Install patches and restart host ${hostId}`)
+        await this.installPatches({ hosts: [host] })
+      } else {
+        log.debug(`Evacuate and restart host ${hostId}`)
```

Comment:
Debug message should be fixed: it's "Restart host" only now.

Reply:
`rebootHost` also evacuates the host first, which is necessary for XS/CH. On XCP-ng, since the host has already been evacuated by `clearHost`, it should only reboot it.